scopes/defender/linter/lint.task.ts

```diff
@@ -1,12 +1,20 @@
 import { BuildTask, BuiltTaskResult, BuildContext, ComponentResult } from '@teambit/builder';
 import { Linter } from './linter';
+import { LinterContext } from './linter-context';
 
 export class LintTask implements BuildTask {
   constructor(readonly aspectId: string, readonly name = 'lint') {}
 
   async execute(context: BuildContext): Promise<BuiltTaskResult> {
     const linter: Linter = context.env.getLinter();
-    const results = await linter.lint(context);
+    const linterContext: LinterContext = Object.assign(
```

Comment:
This would be the same with the spread syntax

```suggestion
    const linterContext: LinterContext = {
      rootDir: context.capsuleNetwork.capsulesRootDir,
      ...context,
    }
```